plugins/DiskSize/plugin.rb

```diff
@@ -18,12 +20,13 @@ def initialize
   end
 
   def store(fqdn)
+
     # Pull node information from cache as a ruby hash
     node_info = @cache.get(fqdn)
 
     # Check for valid data
     if node_info['disk_sizes'].nil? || node_info['disk_sizes'] == 'unknown'
-      raise "No disk_sizes information for #{fqdn}"
+      MyLog.log.warn "DiskSize: No disk_sizes information for #{fqdn}"
```

Comment:
@hsolorza do you know if this line triggers the `rescue` and skips everything below it?
I'm failing some old test because bad data is being inserted into the DB.